src/unix/tty.c

```diff
@@ -30,6 +30,10 @@
 #include <errno.h>
 #include <sys/ioctl.h>
 
+#if defined(__MVS__)
```

Comment:
Perhaps this should be `#if defined(__MVS__) && !defined(IMAXBEL)` in case it gets defined in the future?